packages/manager/apps/dedicated/client/app/ip/ip.controller.js

```diff
@@ -37,6 +38,7 @@ export default /* @ngInject */ function IpMainCtrl(
   $scope.worldPart = coreConfig.getRegion();
   $scope.BRING_YOUR_OWN_IP = BRING_YOUR_OWN_IP;
   $scope.ADDITIONAL_IP = ADDITIONAL_IP;
+  $scope.displayHeaderButtons = !$state.is('app.ip.onboarding');
```

Comment:
this information is already computed in `hasAnyIp` so you can use it directly I guess